t/resty/signals.t

```diff
@@ -0,0 +1,184 @@
+# vi:ft= et ts=4 sw=4
+
+use lib 't/lib';
+use Test::Resty;
+
+plan tests => blocks() * 3;
+
+run_tests();
+
+__DATA__
+
+=== TEST 1: Forward SIGINT to child process
+--- opts: -e 'print(1)'
+--- perlsrc
```

Comment:
Yeah, that's the idea to make `perlsrc` as a `stub/mock_nginx`. `mocked_nginx` sounds
better to me.